### ISSUE TITLE
Insertion Sort in Java

### ISSUE THREAD

**Issue by Chathurangi6** [FOSS-UCSC/FOSSALGO#13]
This issues is about insertion sort in java. https://www.javatpoint.com/insertion-sort-in-java

- [x]  Issue title form Algo/DS Name [Language]

- [x]  I searched or browsed the repo's other issues to ensure this is not a duplicate

 - [x] This Algo/DS is worth to add this repository

 - [x] Assign this issue to me

 - [x]  I will send my PR only after I'm assigned to this issue by any maintainer

**Comment by RavinduSachintha** [comment]
@Chathurangi6 , your contribution was hide due to the renaming of folder. Please if you can make again another pull request for this issue.. Change the file name exactly as "insertionSort.java"
